src/components/Layout/LayoutWrapper.astro

```diff
@@ -0,0 +1,26 @@
+---
+const title = import.meta.env.TITLE
+---
+
+<html lang="en">
+	<head>
+		<meta charset="utf-8" />
+		<meta name="viewport" content="width=device-width" />
+		<meta name="generator" content={Astro.generator} />
+		<link rel="icon" type="image/svg+xml" href="/favicon.svg" />
+		<style>
+			html,
+			body {
+				font-family: system-ui;
+				margin: 0;
+			}
+			body {
+				padding: 2rem;
+			}
+		</style>
+		<title>{title}</title>
+	</head>
+	<body>
+		<slot />
+	</body>
+</html>
```

Comment:
Why is this an `astro` component, not a `svelte` component?

(Also applies to `src/pages/index.astro`)

Reply:
Svelte is just UI framework in this  setup, the pages should be in Astro to make it pages (The layout component is kinda a page too, in svelte it can't be done this way, it should be done via [`svelte:head`](https://svelte.dev/tutorial/svelte-head) ) 
And as for pages - I think we want it to be in astro, to be able to use Astro's `client` and other [directrives](https://docs.astro.build/en/reference/directives-reference/#clientidle)

Reply:
> The layout component is kinda a page too

It's in `src/components` and it's _used_ in a page. Looks like a component to me 🤔

The problem I have with using both `.astro` and `.svelte` components is us having to learn the syntax for both. Should we drop Svelte then? Would using plain Astro be enough for this project?

Reply:
Astro will be enough, but I think we should keep both, it make sense, Svelte is a more UI-based framework, and we can use its advantages, probably not in the first stage, but we can already see [here](https://github.com/brains-and-beards/lingua-franca/pull/3/files#diff-c116be97fd490932495ef3e83845a674bb1bd4f0110ee6626f0c5f777eefa185) how Svelte make things (building UI) easier for us

Reply:
Okay, let's keep Svelte dependency, but try to stick to `.astro` templates and only use `.svelte` when it helps us significantly. That should let us limit the learning curve for this repo.